tests/test_NLM.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+from io import StringIO
+import pandas as pd
+
+from libs.services.NLM import NLM
+
+
+class TestNLM(unittest.TestCase):
+    def setUp(self):
+        self.converter = NLM()
+
+    def test_connect_to_service(self):
+        # test basic NLM service
+        inchikey = 'QNAYBMKLOCPYGJ-REOHCLBHSA-N'
+        args = 'inchikey/equals/{}?data=summary&format=tsv'.format(inchikey)
+        response = self.converter.connect_to_service('NLM', args)
+        self.assertEqual(response.status_code, 200)
+
+        table = pd.read_csv(StringIO(response.text), sep='\t')
+        self.assertFalse(table.empty)
+
+    def test_inchikey_to_name(self):
+        inchikey = 'QNAYBMKLOCPYGJ-REOHCLBHSA-N'
+        name = 'Alanine [USAN:INN]'
+        self.assertEqual(self.converter.inchikey_to_name(inchikey), name)
+
+        inchikey = 'QNAYBMLOXXXXGJ-REOHCLBHSA-N'
+        self.assertIsNone(self.converter.inchikey_to_name(inchikey))
+
+        inchikey = 'QNAYMLGJ-REOLBHSA-N'
+        self.assertIsNone(self.converter.inchikey_to_name(inchikey))
```

Comment:
Looks again like a good opportunity for [parameterized test cases](https://docs.pytest.org/en/6.2.x/example/parametrize.html?highlight=parameters).

Reply:
This (and previous comments) will be targeted in #10.